src/openforms/contrib/objects_api/setup_configuration/tests/test_objects_api_config.py

```diff
@@ -0,0 +1,224 @@
+        self.assertEqual(config1.iot_submission_report, "PDF Informatieobjecttype")
+        self.assertEqual(config1.iot_submission_csv, "CSV Informatieobjecttype")
+        self.assertEqual(config1.iot_attachment, "Attachment Informatieobjecttype")
+
+        self.assertEqual(config2.name, "Config 2")
+        self.assertEqual(config2.identifier, "config-2")
+        self.assertEqual(config2.objects_service, self.objects_service)
+        self.assertEqual(config2.objecttypes_service, self.objecttypes_service)
+        self.assertEqual(config2.drc_service, self.drc_service)
+        self.assertEqual(config2.catalogi_service, self.catalogi_service)
+        self.assertEqual(config2.catalogue_domain, "OTHER")
+        self.assertEqual(config2.catalogue_rsin, "000000000")
+        self.assertEqual(config2.organisatie_rsin, "000000000")
+        self.assertEqual(config2.iot_submission_report, "")
+        self.assertEqual(config2.iot_submission_csv, "")
+        self.assertEqual(config2.iot_attachment, "")
+
+    def test_execute_update_existing_config(self):
+        ObjectsAPIGroupConfigFactory.create(name="old name", identifier="config-1")
+
+        execute_single_step(ObjectsAPIConfigurationStep, yaml_source=CONFIG_FILE_PATH)
+
+        self.assertEqual(ObjectsAPIGroupConfig.objects.count(), 2)
+
+        config1, config2 = ObjectsAPIGroupConfig.objects.all()
+
+        self.assertEqual(config1.name, "Config 1")
+        self.assertEqual(config1.identifier, "config-1")
+        self.assertEqual(config1.objects_service, self.objects_service)
+        self.assertEqual(config1.objecttypes_service, self.objecttypes_service)
+        self.assertEqual(config1.drc_service, self.drc_service)
+        self.assertEqual(config1.catalogi_service, self.catalogi_service)
+        self.assertEqual(config1.catalogue_domain, "TEST")
+        self.assertEqual(config1.catalogue_rsin, "000000000")
+        self.assertEqual(config1.organisatie_rsin, "000000000")
+        self.assertEqual(config1.iot_submission_report, "PDF Informatieobjecttype")
+        self.assertEqual(config1.iot_submission_csv, "CSV Informatieobjecttype")
+        self.assertEqual(config1.iot_attachment, "Attachment Informatieobjecttype")
+
+        self.assertEqual(config2.name, "Config 2")
+        self.assertEqual(config2.identifier, "config-2")
+        self.assertEqual(config2.objects_service, self.objects_service)
+        self.assertEqual(config2.objecttypes_service, self.objecttypes_service)
+        self.assertEqual(config2.drc_service, self.drc_service)
+        self.assertEqual(config2.catalogi_service, self.catalogi_service)
+        self.assertEqual(config2.catalogue_domain, "OTHER")
+        self.assertEqual(config2.catalogue_rsin, "000000000")
+        self.assertEqual(config2.organisatie_rsin, "000000000")
+        self.assertEqual(config2.iot_submission_report, "")
+        self.assertEqual(config2.iot_submission_csv, "")
+        self.assertEqual(config2.iot_attachment, "")
```

Comment:
the name is the only thing you define in the existing record, so the only relevant assertion here seems to be on the name (that the configuration step overwrites it). You already have another test that asserts the other fields getting assigned.

```suggestion
        self.assertEqual(config1.name, "Config 1")
        self.assertEqual(config1.identifier, "config-1")

        self.assertEqual(config2.name, "Config 2")
        self.assertEqual(config2.identifier, "config-2")
```